spec/support/schemas/claims_api/v2/veterans/claims/claim.json

```diff
@@ -132,6 +132,50 @@
       "nullable": true,
       "description": "If true, indicates a decision has been requested and/or a Waiver 5103 has been submitted",
       "example": "false"
+    },
+    "trackedItems": {
+      "type": "array",
+      "description": "",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["closedDate", "description", "dvlpmtTc", "suspensionDate", "requestedDate", "trackedItemId"],
+        "properties": {
+          "closedDate": {
+            "type": "string",
+            "format": "date",
+            "description": "Date the tracked item was closed",
+            "example": "2022-01-01T00:00:00-04:00"
+          },
+          "description": {
+            "type": "string",
+            "description": "Description of the tracked item",
+            "example": "Buddy mentioned - No complete address"
+          },
+          "dvlpmtTc": {
+            "type": "string",
+            "description": "",
```

Comment:
I would like a value for this field, but we don't know what to put there (see [thread](https://oddball.slack.com/archives/C019XKGS1CN/p1660683310668549))

Reply:
It looks like in the latest comment on that thread, Kayla suggests that maybe we can leave off this particular property for now and that we can bring it back if VA.gov confirms they need it.

So let's go ahead and remove it from the JSON schema and the logic that adds it in the response structure for now.